maven-embedder/src/test/java/org/apache/maven/cli/CLIManagerTest.java

```diff
@@ -43,8 +43,10 @@ public void spacedOptions()
     {
         CommandLine cmdLine = cliManager.parse( "-X -Dx=1 -D y=2 test".split( " " ) );
```

Comment:
Also add a test for `-Dz` (implicit `true`)

Reply:
That is covered in the (new) `MavenCliTest#populateProperties`. Should we duplicate it here, then?

Reply:
I would say yes for consistency.

Reply:
This is the last remaining piece.